src/main/java/core/service/FruitStore.java

```diff
@@ -0,0 +1,65 @@
+package core.service;
+
+import core.transactions.BalanceOperationHandler;
+import core.transactions.OperationHandler;
+import core.transactions.PurchaseOperationHandler;
+import core.transactions.ReturnOperationHandler;
+import core.transactions.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStore {
+    private Map<OperationType, OperationHandler> operationHandlers;
+
+    public FruitStore() {
+        operationHandlers = new HashMap<>();
+        operationHandlers.put(OperationType.B, new BalanceOperationHandler());
+        operationHandlers.put(OperationType.S, new SupplyOperationHandler());
+        operationHandlers.put(OperationType.R, new ReturnOperationHandler());
+        operationHandlers.put(OperationType.P, new PurchaseOperationHandler());
```

Comment:
move this hashMap to main method and give it to constructor of your strategy

Reply:
not fixed